waffle/migrations/0004_delete_flag.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.2.7 on 2022-01-21 15:10
```

Comment:
Should we not be replacing this file instead of adding another migration?
https://github.com/PrimarySite/django-waffle/blob/master/waffle/migrations/0002_delete_flag_model.py

Reply:
No, It would be a little risky!
We should manipulate `django_migrations` table and some other stuff if we uncomment it!
Django framework checks the` django_migrations` table and loads modules of any installed app's migrations. it can not detect this missed migration. 
It is recommended not to manipulate the Django system tables since it is a little risky!